migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -21,6 +21,8 @@ With {product-title} 3, administrators individually deployed {op-system-base-ful
 
 {product-title} 4 represents a significant change in the way that {product-title} clusters are deployed and managed. {product-title} 4 includes new technologies and functionality, such as Operators, machine sets, and {op-system-first}, which are core to the operation of the cluster. This technology shift enables clusters to self-manage some functions previously performed by administrators. This also ensures platform stability and consistency, and simplifies installation and scaling.
 
+Beginning with {product-title} 4.13, {op-system} now uses {op-system-base-full} 9.2 packages. This enhancement enables the latest fixes and features as well as the latest hardware support and driver updates. For more information about how this change from RHEL 8.6 to RHEL 9.2 might affect your options configuration and services as well as driver and container support, see the xref:../release_notes/ocp-4-13-release-notes.adoc#ocp-4-13-rhcos-rhel-9-2-packages[RHCOS now uses RHEL 9.2] in the _OpenShift Container Platform 4.13 release notes_.
```

Comment:
I don't know if it makes sense to mention RHEL 8.6 to RHEL 9.2. This document is about migrating from OCP 3 (based on RHEL 7.x or Atomic Host 7.x) to OCP 4.13 (RHEL 9.2). The RHEL 8.6 packages were in OCP 4.11 and OCP 4.12 releases and unrelated to this topic.

Perhaps
```suggestion
Beginning with {product-title} 4.13, {op-system} now uses {op-system-base-full} 9.2 packages. This enhancement enables the latest fixes and features as well as the latest hardware support and driver updates. For more information about how RHEL 9.2 packages might affect your options, configuration, and services as well as driver and container support, see the xref:../release_notes/ocp-4-13-release-notes.adoc#ocp-4-13-rhcos-rhel-9-2-packages[RHCOS now uses RHEL 9.2] in the _OpenShift Container Platform 4.13 release notes_.
```

Reply:
Thanks, Mike! I'll remove the reference to RHEL 8.6.